.github/workflows/ci.yml

```
@@ -245,6 +245,7 @@ jobs:
           -DBOOST_ROOT="%BOOST_ROOT_1_72_0%"                 ^
           -DINSTALL_RUNKRATOS=OFF                            ^
           -DCMAKE_CXX_FLAGS="/Od /we4661 /we4804"            ^
+          -DLOCAL_ZLIB_COMPILATION=ON                        ^
```

```suggestion
          -DFORCE_LOCAL_ZLIB_COMPILATION=ON                        ^
```

can you please also add this in the nightly build?